cmd/server-manager/plugins/events.lua

```diff
@@ -12,20 +16,49 @@ function onEventStart(encodedRaceConfig, encodedServerOpts, encodedEntryList)
     local serverOpts = json.decode(encodedServerOpts)
     local entryList = json.decode(encodedEntryList)
 
+
     -- Uncomment these lines and run the function (start any event) to print out the structure of each object.
     --print("Race Config:", utils.dump(raceConfig))
     --print("Server Options:", utils.dump(serverOpts))
     --print("Entry List:", utils.dump(entryList))
-
+    --print("Countrycodes", utils.dump(countryCodes))
```

Comment:
Is `countryCodes` defined here? It's not loaded until inside `getTrackInfo`. I think it would probably better to call `getTrackInfo` before this point, have it return the race config and server opts, and then call `weatherAPI`. This will also make it easier to implement a change I've suggested further down!

Reply:
You'd need to return countryCodes too, or just don't add it to this dump section, I don't really mind.

Reply:
How this is a remaining line from an old test :)
I call weatherAPI in the getTrackInfo because if it don't find the location, weatherAPI is not used